discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -365,7 +366,7 @@ def update_track_record(
     # if cover_art CID is of a dir, store under _sizes field instead
     if track.cover_art:
         track.cover_art_sizes = track.cover_art
-        track.cover_art = None
+        track.cover_art = None  # type: ignore
```

Comment:
this should be nullable? should be ok to remove ignore

Reply:
Yeah is kinda interesting. In sqlalchemy, this is not the same as None when accessing

<img width="957" alt="Screenshot 2023-08-03 at 7 37 45 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/0cbdec27-5705-481d-a05a-d82fbaf4b4b1">

But interestingly, it will let you construct with it (as optional).

<img width="295" alt="Screenshot 2023-08-03 at 7 40 28 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/0bd2037c-f1d3-4f10-912d-ae427e44d7e9">

Reply:
I think our practice needs to be evolved a bit where we don't rely on mutating records like this? Looking for a better solution...

Reply:
<img width="367" alt="Screenshot 2023-08-03 at 7 45 18 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/30f5a4cc-8801-41af-a28e-85469fb00f25">
sqlalchemy docs suggest doing this, but this just ends up as Any
<img width="301" alt="Screenshot 2023-08-03 at 7 45 37 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/539c078a-51da-4f7a-9859-3df4272e30eb">

Reply:
Re-saving the model file with no changes seemed to fix this... what the?!
<img width="373" alt="Screenshot 2023-08-03 at 7 47 55 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/7578f883-2806-4db4-9a28-eb0e8d583788">